src/views/DamageConsole.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import { get, keys, map } from "lodash";
+import {
+  Box, makeStyles, TextField
```

Comment:
makeStyles & Textfield were unused vars so i guess remove them unless u plan to use them in the future
but i dont think you need a textfield for console

Reply:
removing TextField